system_baseline/openapi/api.spec.yaml

```diff
@@ -382,6 +382,7 @@ components:
       type: object
       required:
         - account
+        - org_id
```

Comment:
can we do this under "required" yet, or do we need an intermediate phase whee it's optional?

Reply:
Also, if yes, then do we need to move account out of required or in another future pass instead?

Reply:
We probably should move `account` from required too to test it with anemic tenants (no `account`, just `org_id`)